templates/tutorialv2/stats/index.html

```diff
@@ -95,7 +95,7 @@ <h2 class="ico-after ico-tutorials">
             {% trans "Aucune statistique a afficher" %}
         {% endif %}
 
-        {% if not display == 'global' %}
+        {% if not display == 'global' and container.parent %}
```

Comment:
is_part or is_chapter? il me semble qu'on a ça.

Reply:
j'avais essayé "not content.is_chapter" retourné false. Donc oui si je mets les deux, j'active seulement si c'est un chapitre